src/Data/Model.php

```diff
@@ -80,4 +81,24 @@ public function toArray(): array
             'topK' => $this->topK,
         ];
     }
+
+    /**
+     * https://ai.google.dev/gemini-api/docs/models/gemini#model-variations
+     */
+    public static function generateGeminiModel(ModelVariation $variation, ?float $generation = null, ?string $version = null): string
```

Comment:
It might be better to keep this method in enum. This Model class actually lists the models returned from the api. Similarly, even though they are both models, this method acts as a helper that returns a string.

Reply:
Moved it to `GeminiHelper` instead.